resources/views/blog.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.main-layout')
+@vite('resources/css/blog.css')
+
+@section('body')
+    @include('partials.nav')
```

Comment:
These two includes are in all your views. This is code duplication for it can be put in the main-layout view.